.github/workflows/snyk.yaml

```diff
@@ -0,0 +1,36 @@
+name: Snyk test Docker image
+on: 
+  push:
+    branches: [master]
+  workflow_dispatch:
+
+jobs:
+  snyk_test:
+    name: Snyk test
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Load openvds image
+      uses: "./.github/actions/load_openvds_image"
+
+    - name: Build docker image
+      run: |
+        tag=snyk_server
+        docker build \
+        -f Dockerfile \
+        --build-arg OPENVDS_IMAGE=${{ env.OPENVDS_IMAGE_TAG }} \
+        --tag $tag \
+        .
+
+    - name: Run Snyk to check Docker image for vulnerabilities
+      id: docker-image-scan
+      continue-on-error: true
+      uses: snyk/actions/docker@master
+      env:
+        SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
```

Comment:
Should we btw be using team default snyk token or should it be service account one? Were there any recommendations about it in the appsec group?

Reply:
There is no proper recommendation, but we need a token for authentication from the Snyk account. Also, its only works with user account token so a security champion from our team can use its token.

Reply:
I created service account when testing myself on my own fork... Seemed fine.
Btw, as far as I understand we are using team token? I don't know what value is there in snyk token variable, but I assumed it would be the team one.